Description: Fixes MANIFEST.in
 Upstream setup.py isn't installing files at all.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-06-07

--- networking-cisco-2015.1.0.orig/MANIFEST.in
+++ networking-cisco-2015.1.0/MANIFEST.in
@@ -7,6 +7,7 @@ include neutron/db/migration/alembic.ini
 include neutron/db/migration/alembic_migrations/script.py.mako
 include neutron/db/migration/alembic_migrations/versions/README
 recursive-include neutron/locale *
+recursive-include networking_cisco *
 
 exclude .gitignore
 exclude .gitreview
